extensions/warp-ipfs/src/store/message/community_task.rs

```diff
@@ -1275,6 +1283,105 @@ impl CommunityTask {
 
         Ok(())
     }
+    async fn process_join_event(&mut self, msg: Message) -> Result<(), Error> {
+        let data = PayloadMessage::<Vec<u8>>::from_bytes(&msg.data)?;
```

Comment:
If we are not going to encrypt the events, we could just have it be `PayloadMessage<CommunityJoinEvents>` instead and skip `serde_json::from_slice`. Thoughts?

Reply:
Just as a note, depending on how the message is sent, we could encrypt the message.

Reply:
Skipped `serde_json::from_slice`, thanks. As discussed, we can leave the join event plaintext here since in the Payload refactor you are working on rn the message will be encrypted there.